client/my-sites/promote-post-i2/utils/index.ts

```diff
@@ -244,6 +244,15 @@ export const formatNumber = ( number: number, onlyPositives = false ): string =>
 	return number.toLocaleString();
 };
 
+export const formatLargeNumber = ( number: number ): string => {
```

Comment:
I searched in Calypso, and I found this function that do a similar thing:
https://github.com/Automattic/wp-calypso/tree/trunk/client/lib/format-number-compact

Do you think we could use that one instead? It might behave differently, which is what you wanted to achieve here.

Reply:
the function is a bit strange.. it wont format millions properly. i tested and I think my method works better. i just added a small improvement to add locale formatting in the number (so for some countries that use comma instead of a point for decimals it will display the result better)